src/components/organisms/WorldGeneralForm/WorldGeneralForm.tsx

```diff
@@ -75,20 +77,18 @@ export const WorldGeneralForm: React.FC<WorldGeneralFormProps> = ({
     setValue,
     reset,
     watch,
-    formState: { dirty, isSubmitting },
     register,
-    errors,
+    control,
     handleSubmit,
   } = useForm<WorldGeneralFormInput>({
     mode: "onSubmit",
     reValidateMode: "onChange",
-    validationSchema: worldStartSchema,
-    validationContext: {
-      creating: !worldId,
-    },
+    resolver: yupResolver(worldStartSchema),
```

Comment:
Part of this `worldStartSchema` is directly tied to the old `validationContext: 
```ts
.when(
    "$creating",
    (creating: boolean, schema: Yup.StringSchema) =>
      creating
        ? schema.test(
            "name",
            "This world slug is already taken",
            testWorldBySlugExists
          )
        : schema
  )
```

Is the schema still triggering and doing the proper check?

Reply:
yes, since [56fdacb](https://github.com/sparkletown/sparkle/pull/2847/commits/56fdacb2463fc94ee442760b996bb3d9ba339539)
Talked about it in the slack
```
hm, so after taking a look at that creating form logic I’m still wondering why it existed
since slugs are unique ( otherwise worlds would overlap and not resolve properly ) we can’t have same slugs no matter whether we create or edit a world
so I’m going to add that validation in each of those cases no matter what form is being edited atm
```